.github/workflows/ci-vulnerability-scans.yml

```diff
@@ -53,13 +61,22 @@ jobs:
       - name: Run Trivy vulnerability scan
         uses: aquasecurity/trivy-action@master
         with:
-          scan-type: "image" # Scan type, image or fs
+          scan-type: image
           image-ref: ${{ steps.build-image.outputs.image }}
-          format: "table" # Can be table, json, sarif, or github
-          exit-code: "1" # Exit code to return if any vulnerabilities are found
-          ignore-unfixed: true # Ignore unpatched/unfixed vulnerabilities
-          vuln-type: "os" # Can be os or library, depending on the type of scan you want to run
-          security-checks: "vuln,secret" # Can be any combo of vuln, secret, and config
+          format: table
+          exit-code: 1
+          ignore-unfixed: true
+          vuln-type: os,library
+          scanners: vuln,secret
+          output: trivy_results.txt
+
+      - name: Save output to workflow summary
+        if: always() # Runs even if there is a failure
+        run: |
+          echo "# Trivy Results" >> $GITHUB_STEP_SUMMARY
+          if [ -f trivy_results.txt ]; then
```

Comment:
same question, is the if statement needed

Reply:
I had it from the filtering so that it wouldn't print the header of the table with no rows, but since we aren't using it now, it isn't necessary